crypto/objects/objects.txt

```diff
@@ -933,11 +933,39 @@ nist_hashalgs 1		: SHA256		: sha256
 nist_hashalgs 2		: SHA384		: sha384
 nist_hashalgs 3		: SHA512		: sha512
 nist_hashalgs 4		: SHA224		: sha224
+nist_hashalgs 5		: id-sha512-224		: sha512-224
+nist_hashalgs 6		: id-sha512-256		: sha512-256
```

Comment:
I agree with @kroeckx on this one - for consistency with the other SHA usage - although I also agree with @dot-asm that it is all over the place in the file so you could pick anything and be "consistent". But the uppercase non-id-form is what I think would be expected here.